packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -1033,6 +1038,10 @@ export function transformCjsImport(
   }
 }
 
+function getIdentifierNameOrLiteralValue(node: Identifier | Literal) {
+  return node.type === 'Identifier' ? node.name : node.value
```

Comment:
Should we do a `String(node.value)` or `node.value as string` here so we don't need to cast to string in the consumers?

Reply:
I thought about it but didn't do it, because the consumer should be aware that it's valid to cast to a string only when they are passing a string literal.